moveit_setup_assistant/src/widgets/robot_poses_widget.h

```diff
@@ -281,5 +281,7 @@ private Q_SLOTS:
 
 }  // namespace moveit_setup_assistant
 
+#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
```

Comment:
Q_DECLARE_METATYPE exists in Qt6 as well: https://doc.qt.io/qt-6/qmetatype.html
Why do you need to ifdef here?

Reply:
There were compiler errors with Qt 6. I switched the `connect`s to modern style connects and there were no errors when removing it so I simply removed the block altogether.